.travis.yml

```diff
@@ -1,38 +1,41 @@
-dist: bionic
+dist: focal
+
+language: minimal
 
 stages:
   - lint
-  - test
+  - unit_test
+  - integration_test
   - deploy
 
 if: branch = master OR type = pull_request
 
-stage: test
-
-name: Project Tests
-
-language: python
-
-python:
-  - "3.8"
-  - "3.9"
-
-install: skip
-
-script:
-  - |
-    set -e
-    make docker/run target=python/test
-    make localstack/up
-    make localstack/pytest
-    make localstack/clean
+before_install:
+  - tmpdaemon=$(mktemp)
+  - sudo jq '."registry-mirrors" += ["https://mirror.gcr.io"]' /etc/docker/daemon.json > "$tmpdaemon"
+  - sudo mv "$tmpdaemon" /etc/docker/daemon.json
+  - sudo systemctl daemon-reload
+  - sudo systemctl restart docker
+  - docker system info
 
 jobs:
   include:
     - stage: lint
       name: Project Syntax Verification
+      script: make docker/run target=lint
+    - stage: unit_test
```

Comment:
We can re-use the same `stage: test` for both jobs, that way they can run in parallel and travis won't take quite so long. Unless you really want to order them.

Reply:
I'm confused.  Does "both jobs" refer to lint and unit_test, or unit_test and integration_test".  If I re-use the same stage, that means I'd need the "set -e" for the script, correct?

Reply:
Oh, no, sorry about that! Travis-ci terminology... A job belongs to a stage. A stage can have multiple jobs. All jobs in a stage may run in parallel. All stages run in sequence. So we can have both jobs, the unit test and the integration test, in the same `test` stage.

Reply:
ok ... it looked like it worked.

Reply:
Oh, I didn't mean to bring the lint stage into this. I thought we were just talking about separating the unit and integration tests into their own jobs. They can share the same stage, something like:

```
    - stage: lint
      name: Project Syntax Verification
      script: make docker/run target=lint
    - stage: test
      name: Run unit tests
      script: make docker/run target=pytest/lambda/tests
    - stage: test
      name: Run integration tests against a mock AWS stack
      install:
        - make docker-compose/install
        - make mockstack/up
      script: make mockstack/pytest/lambda
      after_script:
        - make mockstack/clean
```

I guess we could put the lint job in the test stage also, not the end of the world. I started including it in its own stage just because the lint job is typically much faster than other jobs, and running it first helps to fast fail a PR for quick feedback.

Reply:
I had a lint error in `.travis.yml` in the org-new-account-trust policy repo when I merged stages and the stage sort of hung ... in the "conversations" screen it indicated the Travis job was still working, but in the details it indicated the lint part of the stage had failed.  You can see where that commit has no status symbol assigned to it, so you can't tell if Travis passed or failed on that commit.

Reply:
I'll fix it to isolate the lint, combine the tests, but I need to go to a meetup and I'll be back later.